### PR TITLE
Fix zero value ignored in dictionary generation

### PR DESCRIPTION
**Pull request by kaligrafy**
Add tests for dictionary zero value.

Closes #1644

### CODE REVIEW

File: packages/evolution-generator/src/tests/test_generate_questionnaire_dictionary.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+# License text available at https://opensource.org/licenses/MIT
+
+from scripts.generate_questionnaire_dictionary import process_choices
+
+CHOICES_HEADERS = [
+    "choicesName",
+    "value",
+    "label::fr",
+    "label::en",
+    "label::one::fr",
```
Comment (tahini) [inline packages/evolution-generator/src/tests/test_generate_questionnaire_dictionary.py]: label_one::fr je crois et non label::one::fr

File: packages/evolution-generator/src/tests/test_generate_questionnaire_dictionary.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        assert choices_map["likert5"] == [
+            "-2 : Strongly disagree",
+            "0 : Neutral",
+            "2 : Strongly agree",
+        ]
+
+    def test_skips_rows_without_value(self):
+        rows = choices_data_rows(
+            choices_row(choicesName="yesNo", value="yes", label_en="Yes"),
+            choices_row(choicesName="yesNo", value=None, label_en="Missing value"),
```
Comment (tahini) [inline packages/evolution-generator/src/tests/test_generate_questionnaire_dictionary.py:61]: peut-être ajouter une valeur de false aussi? tant qu'à

**Review by tahini — COMMENTED** [review]
Typo dans la colonne du ::one, je ne sais pas si ça affecte le test ou pas, vu qu'il n'y en a pas de toute façon.

File: packages/evolution-generator/src/tests/test_generate_questionnaire_dictionary.py (hunk 1 above)
Comment (kaligrafy) [reply packages/evolution-generator/src/tests/test_generate_questionnaire_dictionary.py]: fixed

File: packages/evolution-generator/src/tests/test_generate_questionnaire_dictionary.py (hunk 2 above)
Comment (kaligrafy) [reply packages/evolution-generator/src/tests/test_generate_questionnaire_dictionary.py:61]: ajouté

**Review by tahini — APPROVED** [review]
